backend/kesaseteli/common/tests/test_utils.py

```diff
@@ -155,3 +158,11 @@ def test_utc_datetime(args):
     result = utc_datetime(*args)
     assert result == datetime(*args, tzinfo=timezone.utc)
     assert result.tzinfo == timezone.utc
+
+
+@pytest.mark.parametrize("year", [1800, 2022, 2023, 2099])
+def test_get_random_social_security_number_for_year(year):
+    for _ in range(1000):
```

Comment:
another idea could be just loop all days of the year? Not sure how to do it in python but that would be more complete coverage (all possible days) and faster (365/366 items in loop)

Reply:
Well I read the test badly, it was about the testing _random_ ssn